.clang-tidy

```diff
@@ -27,6 +27,7 @@ Checks: >
   -google-readability-casting,
   -google-readability-todo,
   -google-runtime-references,
+  -google-explicit-constructor,
```

Comment:
Why remove this one?

Reply:
Its quite noisy on odb  and I don't agree with all its suggestions.  Throughout dbTypes.h you have constructors like:

dbOrientType(Value orient);

changing that to be explicit is mostly just annoying.  There isn't much risk to such a constructor as the enumerated type prevents misuse from another type.